src/segments/unity.go

```diff
@@ -64,6 +64,7 @@ func (u *Unity) GetUnityVersion() (version string, err error) {
 
 	versionStartIndex := versionPrefixIndex + len(versionPrefix)
 	unityVersion := firstLine[versionStartIndex:]
+	unityVersion = strings.TrimSuffix(unityVersion, "\r")
```

Comment:
```suggestion
	unityVersion = strings.TrimSpace(unityVersion)
```

I suppose we can simply trim all white space, no?

Reply:
Yep, sure can. I lack a lot of familiarity with golang 😅